package proto_ff;

import "yd_fieldoptions.proto";


message noticenotice
{
	optional int32 noticeId = 1[(yd_fieldoptions.field_cname) = "通知ID"];
	optional int32 noitceOnoff = 2[(yd_fieldoptions.field_cname) = "通知开关"];
	optional int32 noticeType = 3[(yd_fieldoptions.field_cname) = "通知类型"];
	optional string openCycle = 4[(yd_fieldoptions.field_cname) = "每周弹出日期", (yd_fieldoptions.field_bufsize) = 32];
	optional string time = 5[(yd_fieldoptions.field_cname) = "弹出时间点", (yd_fieldoptions.field_bufsize) = 32];
	optional int32 levelRequire = 6[(yd_fieldoptions.field_cname) = "玩家等级限定"];
	optional string endDay = 7[(yd_fieldoptions.field_cname) = "开服后多少天不再弹", (yd_fieldoptions.field_bufsize) = 32];
	optional string startDay = 8[(yd_fieldoptions.field_cname) = "开服后多少天开始弹", (yd_fieldoptions.field_bufsize) = 32];
	optional string noticeTitle = 9[(yd_fieldoptions.field_cname) = "通知标题", (yd_fieldoptions.field_bufsize) = 32];
	optional string noticeContent = 10[(yd_fieldoptions.field_cname) = "通知内容", (yd_fieldoptions.field_bufsize) = 63];
}


message Sheet_noticenotice
{
	repeated noticenotice noticenotice_List = 1[(yd_fieldoptions.field_arysize)=116];
}
